egs/multi_zh-hans/ASR/local/compute_fbank_kespeech_dev_test.py

```diff
@@ -52,7 +52,13 @@ def get_parser():
         default=80,
         help="""The number of mel bins for Fbank""",
     )
-
+    # 修改： 添加 指定参数 speed-perturb
```

Comment:
Could you remove the comment or use English instead of Chinese?

Reply:
ok